### PR TITLE
Reduce whitespace on mobile viewports

### PR DESCRIPTION
**Pull request by Johennes**
I'm probably the only person occasionally reading the spec on my phone but whenever I do it's usually quite cumbersome. One of the issues is a sub-optimal use of white space. This PR removes some of the needless emptiness on mobile viewports.

<table>
<tr>
 <td><strong>Before
 <td><strong>After
<tr>
 <td>
 <td>
<tr>
 <td>
 <td>
</table>

Preview: https://pr1770--matrix-spec-previews.netlify.app

### CODE REVIEW

File: assets/scss/_styles_project.scss [hunk 1]
```diff
@@ -471,12 +481,20 @@ of .td-content. This applies the same style to any blockquotes that descend from
...
-  padding-right: 3rem;
+  @media (min-width: 768px) {
+    padding-right: 3rem;
+  }
 }
 
 /* Adjust default styles for info banner */
 .pageinfo-primary {
-  max-width: 80%;
+  @media (min-width: 768px) {
```
[the remark this answers is not in the document]
Comment (richvdh) [reply assets/scss/_styles_project.scss:491]: For that matter, maybe we should use `media-breakpoint-down(md)` for the 768px selectors, but it doesn't really matter.

File: assets/scss/_styles_project.scss [hunk 2]
```diff
@@ -471,12 +481,20 @@ of .td-content. This applies the same style to any blockquotes that descend from
...
+    max-width: 80%;
```
Comment (richvdh) [inline assets/scss/_styles_project.scss:492]: No action required, but https://github.com/matrix-org/matrix-spec/issues/761 is related here. The whole thing is a mess.

Comment (Johennes) [reply assets/scss/_styles_project.scss:492]: Oh, interesting. Yeah, I had noticed some of the other content having odd widths, too, but tried to keep changes to a minimum here.

File: assets/scss/_styles_project.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (Johennes) [reply assets/scss/_styles_project.scss]: Good idea and it were only three call sites anyway, so I took care of that.

**Review by richvdh — APPROVED** [review]
Thank you!

**Pull request merged**